IoTCoreDefaultApp/IoTCoreDefaultApp/Utils/CortanaHelper.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Threading.Tasks;
+using Windows.Foundation;
+using Windows.Services.Cortana;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Data;
+
+namespace IoTCoreDefaultApp
+{
+    class CortanaHelper
+    {
+        public static void LaunchCortanaToConsentPageAsyncIfNeeded()
+        {
+            // Do nothing for devices that do not support Cortana
+            if (CortanaSettings.IsSupported())
```

Comment:
Using CortanaSettings WinRT API needs to be checked with IsAPIPresent or IsContractPresent checks